boards/mulle/include/periph_conf.h

```diff
@@ -128,28 +128,44 @@ static const uart_conf_t uart_config[] = {
 /** @} */
 
 /**
- * @name ADC configuration
+ * @name    ADC configuration
  * @{
  */
 static const adc_conf_t adc_config[] = {
-    /* dev, pin, channel */
-    [ 0] = { ADC1, GPIO_UNDEF, 26 },       /* internal: temperature sensor */
-    [ 1] = { ADC1, GPIO_UNDEF, 27 },       /* internal: band gap */
-    [ 2] = { ADC1, GPIO_UNDEF, 29 },       /* internal: V_REFSH */
-    [ 3] = { ADC1, GPIO_UNDEF, 30 },       /* internal: V_REFSL */
-    [ 4] = { ADC1, GPIO_UNDEF, 23 },       /* internal: DAC0 module output level */
-    [ 5] = { ADC1, GPIO_UNDEF, 18 },       /* internal: VREF module output level */
-    [ 6] = { ADC1, GPIO_UNDEF,  0 },       /* on board connection to Mulle Vbat/2 on PGA1_DP pin */
-    [ 7] = { ADC1, GPIO_UNDEF, 19 },       /* on board connection to Mulle Vchr/2 on PGA1_DM pin */
-    [ 8] = { ADC0, GPIO_UNDEF,  0 },       /* expansion port PGA0_DP pin */
-    [ 9] = { ADC0, GPIO_UNDEF, 19 },       /* expansion port PGA0_DM pin */
-    [10] = { ADC1, GPIO_PIN(PORT_A, 17), 17 }, /* expansion port PTA17 */
-    [11] = { ADC1, GPIO_PIN(PORT_B,  0),  8 }, /* expansion port PTB0  */
-    [12] = { ADC0, GPIO_PIN(PORT_C,  0), 14 }, /* expansion port PTC0  */
-    [13] = { ADC1, GPIO_PIN(PORT_C,  8),  4 }, /* expansion port PTC8  */
-    [14] = { ADC1, GPIO_PIN(PORT_C,  9),  5 }, /* expansion port PTC9  */
-    [15] = { ADC1, GPIO_PIN(PORT_C, 10),  6 }, /* expansion port PTC10 */
-    [16] = { ADC1, GPIO_PIN(PORT_C, 11),  7 }, /* expansion port PTC11 */
```

Comment:
The reason I added the indices in the first place is that this list is too long to immediately see which adc line to use in the applications. I would like to keep the explicit indices for readability.